digdag-cli/src/main/java/io/digdag/cli/client/EnableSchedule.java

```diff
@@ -15,10 +16,15 @@
 public class EnableSchedule
         extends ClientCommand
 {
+    @Parameter(names = {"-m", "--mode"})
+    String modeString = null;
+
     @Override
     public SystemExitException usage(String error)
     {
         err.println("Usage: " + programName + " enable <schedule-id> | <project-name> [name]");
+        err.println("  Options:");
+        err.println("    -m, --mode MODE                  update schedules with this mode for skipping sessions (default: none)");
```

Comment:
It is better to something explain about MODE. In addition we may need document for it.

Reply:
Ditto.